src/Avalonia.Controls/SplitView.cs

```diff
@@ -258,6 +260,7 @@ public SplitViewPanePlacement PanePlacement
         /// <summary>
         /// Gets or sets the Pane for the SplitView
         /// </summary>
+        [DependsOn(nameof(PaneTemplate))]
```

Comment:
I don't think this attribute actually does anything currently, but worth keeping it here as a reminder to implement this (or something like it) one day!